### PR TITLE
Cig pack and matchbox tweaks

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Screenshot_1

**Changelog**

:cl:
- tweak: Cigarette pack and matchbox sizes were downed to 5
- tweak: Matchbox inventory size was adjusted

### CODE REVIEW

**Comment by 0x6273** [comment]
Containers shouldn't have bigger capacity than their size. It's a matchbox, not a TARDIS.

File: Resources/Prototypes/Entities/Objects/Tools/matches.yml [hunk 1]
```diff
@@ -72,6 +72,14 @@
   - type: Item
     sprite: Objects/Tools/matches.rsi
     heldPrefix: matchbox
+    size: 5
+  - type: Storage
+    capacity: 6
+    whitelist:
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Tools/matches.yml:78]: literally 1984 we cant put nuke disk (somewhat reasonable) or pens (very reasonable) in matchbox

**Comment by lapatison** [comment]
> Containers shouldn't have bigger capacity than their size. It's a matchbox, not a TARDIS.

Undersized whitelisted containers already presented in game, and I see nothing bad in here.
Matchbox is not a TARDIS, but previously you were able to put a crowbar in here.
I could have made it size 6, like cig pack, but I annoyed by the fact that in the game where objects sizes are usually multiple of five (5,10,etc.), and containers are multiple of ten (10 (like coat pocket),30,100) cig pack have this 1 of additional size.

For some reason, I thought that reducing the amount of cigs\matches down to 5 will be more unpopular decision.
Perhaps, I should actually do this? @deltanedas Simultaneously with whitelist removal.
5 size 5 container?

**Comment by 0x6273** [comment]
> Matchbox is not a TARDIS, but previously you were able to put a crowbar in here.

No, a crowbar is size 10 so it can't fit in a matchbox.

> 5 size 5 container?

Seems like a pointless change. Size/capacity doesn't need to be a multiple of 5. There's nothing here that needs fixing.

**Comment by lapatison** [comment]
> No, a crowbar is size 10 so it can't fit in a matchbox.
> 
> >

 Matchbox is 10 size \10 container right now. You can see it in the code.

>  Seems like a pointless change. Size/capacity doesn't need to be a multiple of 5. There's nothing here that needs fixing.
> 
> >

Not a pointless change, but QoL fix.

**Comment by asperger-sind** [comment]
> No, a crowbar is size 10 so it can't fit in a matchbox.

Matchboxes have a storage capacity of 10 whilst crowbars have a storage value of 10; you can fit crowbars into matchboxes.

> > 5 size 5 container?
> 
> Seems like a pointless change. Size/capacity doesn't need to be a multiple of 5. There's nothing here that needs fixing.

it doesn't HAVE to be a multiple of 5, but it is nice if it is; running around with 96/100 capacity in your backpack and being unable to put - for example - an epipen into it sucks. I think this change is fine balance-wise and a nice thing QoL-wise.

**Comment by 0x6273** [comment]
> Matchboxes have a storage capacity of 10 whilst crowbars have a storage value of 10; you can fit crowbars into matchboxes.

My bad, I thought they were 6 since they have 6 match sticks. It should probably be changed to 6 size and capacity then so it's the same as cig packs.

> it doesn't HAVE to be a multiple of 5, but it is nice if it is; running around with 96/100 capacity in your backpack and being unable to put - for example - an epipen into it sucks. I think this change is fine balance-wise and a nice thing QoL-wise.

The medipens are size 3.

File: Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml [hunk 2]
```diff
@@ -14,8 +14,13 @@
...
   - type: Storage
     capacity: 6
+    whitelist:
+      tags:
+      - Cigarette
+      - Cigar
+      - Matchstick
   - type: Item
-    size: 6
+    size: 5
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:23]: Please don't make size less than capacity, the item size situation is already bad enough.

This also can't fit basic stuff like pens or joints or whatever?

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml (hunk 2 above)
Comment (lapatison) [reply Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:23]: Currently, its whitelsited to cigs, cigars, and matches. Tahts why I made size less capacity.
Ill make it 5\5 no whitelist then

File: Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml [hunk 3]
```diff
@@ -14,8 +14,13 @@
   - type: SpaceGarbage
   - type: Storage
     capacity: 6
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml:16]:
```suggestion
    capacity: 5
```

File: Resources/Prototypes/Entities/Objects/Tools/matches.yml [hunk 4]
```diff
@@ -71,6 +71,14 @@
   - type: Item
     sprite: Objects/Tools/matches.rsi
     heldPrefix: matchbox
+    size: 5
+  - type: Storage
+    capacity: 6
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Tools/matches.yml:76]:
```suggestion
    capacity: 5
```

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**